components/popover/index.js

```diff
@@ -219,6 +220,7 @@ class Popover extends Component {
 			'is-' + xAxis,
 			{
 				'is-mobile': isMobile,
+				'no-arrow': noArrow,
```

Comment:
This is not very consistent with [modifier class naming](https://github.com/WordPress/gutenberg/blob/master/docs/reference/coding-guidelines.md#naming). I might have preferred something like `has-no-arrow` or `is-without-arrow` or `is-no-arrow`.

Reply:
I like how you see a comment from Andrew from time to time on old PRs :P 

Anyway, I just pushed a fix to master (accidentally, I thought I was on a branch 😬 )

Reply:
https://github.com/WordPress/gutenberg/commit/3ce3feea8b0f2c81df3aee82e8a450466345b786

Reply:
I enjoy to surf the `git blame` 😄